docker/README.md

```diff
@@ -12,6 +12,13 @@ You should refer to the .Dockerfile for the actual list. At the time of editing,
 - node-template
 - chain-spec-builder
 
+> Warning: Currently the pre-built image [`parity/substrate:latest`](https://hub.docker.com/layers/paritytech/substrate/latest/images/sha256-d1be27ff2a93d7de49a5ef9449b4e7aa5f479d9d03f808ec34bf2e8cea89cdc4?context=explore) is outdated and uses `substrate 3.0.0-dev-ea387c63471`. The entrypoint it uses is `ENTRYPOINT ["/usr/local/bin/substrate"]` so it only supports running that old Substrate binary and to use the image you need to provide options to it in the Docker run command but without passing the Substrate binary (i.e. `docker run --rm -it parity/substrate --version`).
```

Comment:
What is that conveying? 

I think we should remove this.

Reply:
yes if the user follows the instructions and runs `./build.sh` before running the `docker run ...` commands then it'll build an image and a container with the latest substrate.

but if they forget to run the `./build.sh` step (by not following the instructions, although that's unlikely..) then it'll fetch that old substrate image parity/substrate:latest from docker hub that is 3 years old, and they may not realise they're not using the latest version until it doesn't function as intented.  

do you think it would be worth perhaps changing it to `> Note: If you do not first run the build.sh script to generate the parity/substrate image before using the `docker run` command to create a container based on that image name, then it will instead fetch a pre-generated version of that image that may not be the latest version.`? or just remove it?

is there a reason why that parity/substrate:latest doesn't have the latest version like the image ci-linux:production?

Reply:
i've pushed a commit removing the warning, i think it's sufficient to expect users to follow the instructions